From 89bfb6b05852156ebd1e897abac91669935a6589 Mon Sep 17 00:00:00 2001
From: "Brian W. Hart" <hartb@us.ibm.com>
Date: Wed, 13 May 2020 19:44:02 +0000
Subject: [PATCH] Relax tolerance of some unit tests

A couple of the unit tests fall just out of tolerance on Power.
Relax the tolerance slightly to allow them to pass.
---
 allennlp/tests/modules/attention/additive_attention_test.py | 2 +-
 allennlp/tests/modules/similarity_functions/linear_test.py  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/allennlp/tests/modules/attention/additive_attention_test.py b/allennlp/tests/modules/attention/additive_attention_test.py
index ee8c8cc..8b50179 100644
--- a/allennlp/tests/modules/attention/additive_attention_test.py
+++ b/allennlp/tests/modules/attention/additive_attention_test.py
@@ -27,4 +27,4 @@ class TestAdditiveAttention(AllenNlpTestCase):
         assert result.shape == (2, 4)
         assert_almost_equal(result, [
                 [1.975072, -0.04997836, 1.2176098, -0.9205586],
-                [-1.4851665, 1.489604, -1.890285, -1.0672251]])
+                [-1.4851665, 1.489604, -1.890285, -1.0672251]], decimal=6)
diff --git a/allennlp/tests/modules/similarity_functions/linear_test.py b/allennlp/tests/modules/similarity_functions/linear_test.py
index dd5b88c..3ff90fa 100644
--- a/allennlp/tests/modules/similarity_functions/linear_test.py
+++ b/allennlp/tests/modules/similarity_functions/linear_test.py
@@ -24,7 +24,7 @@ class TestLinearSimilarityFunction(AllenNlpTestCase):
         b_vectors = torch.FloatTensor([[[0], [1]]])
         result = linear(a_vectors, b_vectors).data.numpy()
         assert result.shape == (1, 2,)
-        assert_almost_equal(result, [[2.3, -1.1]])
+        assert_almost_equal(result, [[2.3, -1.1]], decimal=6)
 
     def test_forward_works_with_higher_order_tensors(self):
         linear = LinearSimilarity(7, 7, combination='x,y')
-- 
1.8.3.1

